src/main/java/org/embulk/input/marketo/rest/MarketoBaseRestClient.java

```diff
@@ -96,6 +107,13 @@ private String requestAccessToken()
         params.put("client_id", clientId);
         params.put("client_secret", clientSecret);
         params.put("grant_type", "client_credentials");
+
+        // add partner api key to the request
+        if (partnerApiKey.isPresent()) {
+            LOGGER.info("> Request access_token with partner_id: {}", StringUtils.abbreviate(partnerApiKey.get(), 8));
```

Comment:
None blocking but we should be careful to log out partnerApiKey event just 8 chars

Reply:
It's 5 characters in fact. IMO it's no issue because the key is used for statistic and troubleshooting when an issue occur. It's not replacing of client credentials